tests/typeConverter.test.ts

```diff
@@ -932,3 +932,157 @@ export type UserOperation = 'Read' | 'Write';
 `;
     expect(resultString).toEqual(expectedString);
 });
+
+it('should return overrided enum schema', async () => {
+
+    // What will be fetched from Swagger Json
+    const example = {
+        components: {
+            schemas: {
+                ServiceOfferKind: {
+                    type: 'string',
+                    description: '',
+                    'x-enumNames': [
+                        'MasteringAndDistribution',
+                        'Video',
+                        'Samples',
+                        'Mastering',
+                        'Distribution',
+                        'Sessions',
+                    ],
+                    enum: ['MasteringAndDistribution', 'Video', 'Samples', 'Mastering', 'Distribution', 'Sessions'],
+                },
+            },
+        },
+    };
+
+    const resultString = parseSchemas({
+        json: example,
+        swaggerVersion: 3,
+        // Overrided value "ServiceOfferKind" enum
+        overrideSchemas: [
+            {
+                ServiceOfferKind: {
+                    type: 'string',
+                    description: 'Warning! This type is overrided',
+                    enum: ['masteringAndDistribution', 'video', 'samples', 'mastering', 'distribution', 'sessions'],
+                },
+            },
+        ],
+    });
+
+    const expectedString = `/**
+ * Warning! This type is overrided 
+ */
+export type ServiceOfferKind = 'masteringAndDistribution' | 'video' | 'samples' | 'mastering' | 'distribution' | 'sessions';
```

Comment:
So this is not an enum but a union type

Reply:
@ardeois yes, correct. But for Swagger it is an enum